### PR TITLE
blog: add "the understanding urbit podcast" post

### PR DESCRIPTION
**Pull request by urcades**
what it says on the tin

### CODE REVIEW

File: content/blog/the-understanding-urbit-podcast.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ship = "~satsyt-sogleb"
+image ="https://media.urbit.org/site/posts/essays/uu-podcast.png"
++++
+
+<br>
+
+<img class="ba" src="https://media.urbit.org/site/posts/essays/uu-podcast.png">
+
+The Understanding Urbit Podcast
+~satsyt-sogleb
```
Comment (matildepark) [inline content/blog/the-understanding-urbit-podcast.md]: Cut this, as we already have top matter above this

File: content/blog/the-understanding-urbit-podcast.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+Over the course of seven episodes the interviews explore subjects ranging from the philosophy of technology to the performance optimization of Nock execution. Most are non-technical, but even the technical ones will be understandable and entertaining to a non-technical audience. That latter fact is the result of just how much fun the Tlon team is to be around.
+
+The series can be found on iTunes, Spotify, and most other streaming platforms.
+
+<iframe style="border: none" src="//html5-player.libsyn.com/embed/episode/id/13669184/height/90/theme/custom/thumbnail/yes/direction/backward/render-playlist/no/custom-color/000000/" height="90" width="100%" scrolling="no"  allowfullscreen webkitallowfullscreen mozallowfullscreen oallowfullscreen msallowfullscreen></iframe>
+
+We begin our journey with Urbit veteran Galen Wolfe-Pauley who introduces the project and its design principles while reflecting on the collective Stockholm syndrome we suffer in the grasp of existing computing models. We learn a bit about Galen’s background in architecture, and the importance of invisible technology.
```
Comment (matildepark) [inline content/blog/the-understanding-urbit-podcast.md]: Pauly

**Comment by crrncllns** [comment]
Lgtm

File: content/blog/the-understanding-urbit-podcast.md (hunk 1 above)
Comment (urcades) [reply content/blog/the-understanding-urbit-podcast.md]: Good catch

File: content/blog/the-understanding-urbit-podcast.md (hunk 2 above)
Comment (urcades) [reply content/blog/the-understanding-urbit-podcast.md]: fixed

**Review by matildepark — APPROVED** [review]

**Pull request merged**